airgun/entities/subscription.py

```diff
@@ -48,12 +48,13 @@ def has_manifest(self):
         May be None if user can't verify reliably if manifest is
         uploaded or not due to missing permissions
         """
-        view = self.navigate_to(self, 'All')
         try:
-            view.add_button.wait_displayed()
+            view = self.navigate_to(self, 'Manage Manifest')
+            result = "http" in view.manifest.red_hat_cdn_url.value
+            view.close_button.click()
```

Comment:
Some concern about calling this close_button inside the try here, instead of in a `finally` block, what does it leave on the displayed page if there is an exception (especially other than a TimedOutError) when reading `red_hat_cdn_url.value`?